tools/wptserve/wptserve/server.py

```diff
@@ -378,8 +378,10 @@ def handle_one_request(self):
                 self.close_connection = True
                 for stream_id, (thread, queue) in stream_queues.items():
                     queue.put(None)
+        except Exception as e:
+            self.logger.debug('(%s) ERROR - Unexpected Error - \n%s' % (self.uid, str(e)))
```

Comment:
Why isn't this just `self.logger.error` ?

Reply:
oh yes that would make more sense...

Reply:
Updated this, and also changed the other exception catch to `self.logger.error` aswell